### PR TITLE
fix(): when using s3, parse stream output to buffer for html files

### PR DESCRIPTION
**Pull request by jithine**
## Objective

1. increase token timeout to 13hrs
2. map yidf/state to txt extension
3. make sure html files can be rendered when using s3

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: bin/server [hunk 1]
```diff
@@ -47,7 +47,7 @@ async function init() {
 
         return logger.info('Server running at %s', server.info.uri);
     } catch (err) {
-        logger.error(err);
+        logger.error(err.toString());
```
Comment (adong) [inline bin/server:50]: How about `JSON.stringify`, does JSON.stringify capture many info nosies?

Comment (jithine) [reply bin/server:50]: this is a bug with logger we use `winston`  This is what we should do long term https://github.com/winstonjs/winston/issues/1338#issuecomment-843793731

File: helpers/mime.js [hunk 2]
```diff
@@ -2,13 +2,19 @@
...
+};
+
 /**
  * getMimeFromFileExtension
  * @param  {String} fileExtension  File extension (e.g. css, txt, html)
  * @return {String} text/html
  */
 function getMimeFromFileExtension(fileExtension) {
-    return mime.lookup(fileExtension) || '';
+    return mime.lookup(fileExtension) || mime.lookup(FORCE_EXTENSION_MAPPING[fileExtension]) || '';
```
Comment (adong) [inline helpers/mime.js:17]: I think the 2nd condition should happen early.

```
mime.lookup(FORCE_EXTENSION_MAPPING[fileExtension] || fileExtension) || ''
```

File: helpers/aws.js [hunk 3]
```diff
@@ -31,6 +32,10 @@ class AwsClient {
             }
         };
 
+        if (config.sessionToken) {
```
Comment (pritamstyz4ever) [inline helpers/aws.js:35]: is this added for local debugging only?

Comment (jithine) [reply helpers/aws.js:35]: Yes, any setup with a federated login, not sure if there is a scope to use this to move away from access key based login.

**Review by pritamstyz4ever — APPROVED** [review]

File: helpers/aws.js (hunk 3 above)
Comment (pritamstyz4ever) [reply helpers/aws.js:35]: Added this, to remove IAM user dependency  https://github.com/screwdriver-cd/screwdriver/issues/2454

**Pull request merged**